x-pack/test/functional/apps/visualize/reporting.ts

```diff
@@ -52,7 +53,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('becomes available when saved', async () => {
-        await PageObjects.reporting.setTimepickerInDataRange();
```

Comment:
The time range set by this function, but I didn't want this test to be impacted. So I changed it to use the timePicker service directly.